_docs/_latest/develop/dev-setup.md

```diff
@@ -173,32 +118,43 @@ You can find the `CPPFLAGS` and `LDFLAGS` information and export them as follows
     CPPFLAGS: -I/SOME/PATH/TO/openssl/include
   ```
 
-Then copy the `LDFLAGS` and `CPPFLAGS` values and export them, similar to the 
-following (use the values from your terminal, not "`/SOME/PATH/TO`"):
+Next, copy the `LDFLAGS` and `CPPFLAGS` values and export them, similar to the 
+following:
 
   ```bash
   $ export CPPFLAGS=-I/SOME/PATH/TO/openssl/include
   $ export LDFLAGS=-L/SOME/PATH/TO/openssl/lib
   ```
+In the above example, `/SOME/PATH/TO` represents the path specific to your
+system. Make sure to use the values from your terminal.
 
 ### Configuring Forseti settings
 
-Before you run Forseti, you need to edit the forseti_conf.yaml file, found in
-`forseti-security/configs/forseti_conf.yaml`. Refer to 
-["Configuring Forseti"]({% link _docs/latest/configure/configuring-forseti.md %}) 
+Before you run Forseti, you need to edit the forseti configuration file. 
```

Comment:
The linked page no longer exists. Can we update to:

`Before you run Forseti, follow the process in the [Configuring Forseti](link) guide to edit <filename>.`

With that change I think we'll be fine. I'll approve the PR so you can update and submit at your leisure, as I'm about to be offline.